src/System.Xml.XDocument/tests/events/System.Xml.XDocument.Events.Tests.csproj

```diff
@@ -9,7 +9,6 @@
     <AssemblyName>System.Xml.XDocument.Events.Tests</AssemblyName>
     <SignAssembly>false</SignAssembly>
     <ProjectGuid>{C560E194-5B14-4112-ABC6-3208491E53E6}</ProjectGuid>
-    <TestCategories>OuterLoop</TestCategories>
```

Comment:
I think it is too late for that: https://github.com/dotnet/buildtools/pull/144
This pretty much reverted all my changes.